stories/utils/adsense-data-zeroing.js

```diff
@@ -1,3 +1,29 @@
+/**
+ * AdSense Settings components.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
```

Comment:
Please update the year to 2022 :)

```suggestion
 * Site Kit by Google, Copyright 2022 Google LLC
```